### DISCUSSION TITLE
Inventory Put/Post failing with straight forward response

### DISCUSSION THREAD

**Discussion by fiqjeff** [Sellbrite/Sellbrite-API#13] [Q&A] [answered]
Hello,

I am working with the inventory endpoint and I submitted the necessary details in the payload.

However, whenever i call the PUT or even the POST endpoint. I get this error.
"{\"error\":\"Warehouse not found\"}"

This is a very straight forward response, but I'm confused because i'm passing in the correct UUID for the warehouse that was fetched from the warehouse get endpoint.

The response I get back from warehouse get endpoint is this, ignore the $ as i've replaced our actual address values with $.
	{
		"uuid": "2be7387c-4260-460f-8c42-c6f83c606b0c",
		"name": "$$$$$$$$",
		"inventory_master": "sellbrite",
		"address_1": "$$$$$$",
		"address_2": null,
		"city": "$$$$$",
		"state_region": "$$",
		"postal_code": "$$$$-$$$$",
		"country_code": "US",
		"archived": false
	}

So then when I created the PUT request like so...
{
  "sku": "SOMESKU",
  "description": "SOME DESCRIPTION",
  "warehouse_uuid": "2be7387c-4260-460f-8c42-c6f83c606b0c",
  "on_hand": 10,
  "available": 10,
  "bin_location": "10",
  "cost": 20
}

It returned with the straight forward error about the warehouse_uuid.
Is there some sort of setting that needs to be turned on for the warehouse?

Again i also tried the POST method with the 3 parameters and both returned the same error.
Thanks,

Jeff

**Comment by hlee-godaddy** [comment, marked as answer]
Hi Jeff,

Thanks for reaching out!

Please try assigning the params to `inventory` and wrapping it in the request body. For example:
```
{
  inventory: {
    sku: 'SOMESKU',
    description: 'SOME DESCRIPTION',
    warehouse_uuid: '2be7387c-4260-460f-8c42-c6f83c606b0c',
    on_hand: 10,
    available: 10,
    bin_location: '10',
    cost: 20,
  },
};
```
For your reference: https://developer.sellbrite.com/endpoints/inventory-put

Let us know if this helps.

**Reply by fiqjeff** [reply]
Ok great, so the error was a little misleading.

That did the trick thank you.
